README.rst

```diff
@@ -1,3 +1,23 @@
+.. image:: https://img.shields.io/pypi/v/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
+
+.. image:: https://img.shields.io/pypi/l/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
+
+.. image:: https://img.shields.io/pypi/wheel/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
+
+.. image:: https://img.shields.io/pypi/pyversions/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
```

Comment:
Yup. I would prefer to remove this badge. This information should be maintained in the `setup.py` trove classifiers, so adding yet another place to update is just more work for maintainers.